CHANGELOG.md

```diff
@@ -12,6 +12,20 @@
 
 ### FEATURES
 
+- [Forked gaia](https://github.com/atomone-hub/atomone/commit/d49b86344c3ee42f5182278601c6ce2bd1eff48e): Forked the gaia codebase and renamed it to AtomOne
+- [Removed components](https://github.com/atomone-hub/atomone/commit/9e75d78bd6adc490acee869ac98217a1623a9c6d): Removed ibc, ibc-middleware, ica and ics
+- [Removed x/metaprotocols module](https://github.com/atomone-hub/atomone/commit/8cc9a02587c96f819d346673e40b4b683f3c0f5b): Removed x/metaprotocols module
+- [Forked cosmos-sdk/x/gov](https://github.com/atomone-hub/atomone/commit/ca0724f036f077ffd3b2efc2a43db2ed98ad885e): Forked x/gov
+- [#5](https://github.com/atomone-hub/atomone/pull/5): Disabled validator vote inheritance
+- [#7](https://github.com/atomone-hub/atomone/pull/7): Removed usage of v1.ValidatorGovInfo
+- [#8](https://github.com/atomone-hub/atomone/pull/8): Added x/gov/types
+- [#9](https://github.com/atomone-hub/atomone/pull/9): Remove No With Veto, and set default pass threshold to 2/3 and quorum to 25%
+- [#10](https://github.com/atomone-hub/atomone/pull/10): Changed min voting period to 3 weeks
+- [#11](https://github.com/atomone-hub/atomone/pull/11): Added constitution amendment and law proposals with specific quorum and pass thresholds
+- [#13](https://github.com/atomone-hub/atomone/pull/12): Added late quorum voting period extension
+- [#13](https://github.com/atomone-hub/atomone/pull/13): Added a minimum amount per deposit
+- [#15](https://github.com/atomone-hub/atomone/pull/15): Added constitution to x/gov
```

Comment:
Nice, it's more complete than my attempt in #18. One tiny thing is about the format, usually it's the text first, then the link to the PR. I think we should stick to this.

Reply:
yeah I think aside from @tbruyelle comment this looks pretty good! TY

Reply:
@giunatale I reformatted the entries, should be ready to merge now.